price_parser/parser.py

```diff
@@ -36,15 +37,33 @@ def fromstring(cls, price: Optional[str],
         which may contain currency, as a hint. If currency is present in
         ``price`` string, it could be **preferred** over a value extracted
         from ``currency_hint`` string.
+
+        ``decimal_separator`` is optional; it is used to determine the
+        decimal separator in price. If ``decimal_separator`` is ``None``,
+        then it is guessed from ``price`` string. If ``decimal_separator``
+        is ``"."``, then ``1.000`` is parsed as ``1``. If it is ``,```,
+        then ``1.000`` is parsed as ``1000``.
+
+        ``digit_group_separator`` is optional; it is used to determine the
+        digit group separator in price. If ``digit_group_separator`` is
+        ``None``, then it is guessed from ``price`` string. If
+        ``digit_group_separator`` is ``"."``, then ``1.000`` is parsed as
+        ``1000``. If it is ``,``, then ``1.000`` is parsed as ``1``.
         """
+        currency = extract_currency_symbol(price, currency_hint)
+        if currency is not None:
+            currency = currency.strip()
+        if currency in CURRENCIES_DEC_DIGITS:
+            if CURRENCIES_DEC_DIGITS[currency] == 3:
+                if not digit_group_separator:
+                    digit_group_separator = ','
+                if not decimal_separator:
+                    decimal_separator = '.'
```

Comment:
Why are we assuming that if a currency has 3 decimal digits it uses `.` as decimal separator and `,` as digit group separator, is that a known fact? Also, `digit_group_separator` seems unused.

Reply:
>  Also, digit_group_separator seems unused.

Yes, I wasn't seeing how to use it. Now I think that if we have `digit_group_separator` maybe we just remove it. Would that be a good way to use it?

> I thought the currency data included something to indicate which separators to use.

On currency we have info about the amount of decimals the currency has, I was thinking on using that to know if the separator is a decimal or a group separator (I'm using CURRENCIES_DEC_DIGITS for that). The thing is how to manage the case where there are both. 
example: 123,456.789

Reply:
> Now I think that if we have `digit_group_separator` maybe we just remove it. Would that be a good way to use it?

Sounds good to me, yes.

> The thing is how to manage the case when they are both present.

Indeed, and it is no small thing. I think it is OK not to try and guess the right separators from the currency, and only give the option to specify a `digit_group_separator` to correct bad parsings.